external/anomaly/anomaly_classification/configs/padim/configuration.py

```diff
@@ -0,0 +1,46 @@
+# Copyright (C) 2021 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions
+# and limitations under the License.
+
+from attr import attrs
+from ote_sdk.configuration.elements import (ParameterGroup,
+                                            add_parameter_group,
+                                            string_attribute,
+                                            selectable)
+
+from anomaly_classification.configs.configuration import AnomalyClassificationConfig
+from anomaly_classification.configs.configuration_enums import ModelName
+
+
+@attrs
+class PadimConfig(AnomalyClassificationConfig):
```

Comment:
`PadimConfig` should be named as `AnomalyClassificationConfig`. This is because of potential mypy issues we will face in `task.py`.

Reply:
I'm not sure about this. In the task we could just use the `BaseAnomalyClassificationConfig` typing, because the task would never need access to any model-specific parameters.

Reply:
In terms of typing yes, but in case we need any task-related parameter, we would get errors. But I agree, this could stay as is now since this integration already takes quite some time. better finish sooner